app/app.go

```diff
@@ -496,7 +496,7 @@ func wireValidatorMock(conf Config, pubshares []eth2p0.BLSPubKey, sched core.Sch
 				eth2http.WithTimeout(time.Second*10), // Allow sufficient time to block while fetching duties.
 			)
 			if err != nil {
-				log.Warn(ctx, "Cannot connect to validatorapi", z.Err(err))
+				log.Warn(ctx, "Cannot connect to validatorapi", err)
 				return
```

Comment:
most `log.Warn` had `z.Err(err)`, which caused inconsistent logging and resulted in inconsistent API